spec/services/listing_service/api/shapes_spec.rb

```diff
@@ -26,6 +26,7 @@
               { locale: "en", name: "Selling", action_button_label: "Buy" },
               { locale: "fi", name: "Myydään", action_button_label: "Osta" }
             ],
+            url_source: "Selling",
```

Comment:
It's pretty confusing variable name - since it's not url but just a word that can be added to url.
So, I thought that we add this to Shape entity, when we know how to use it over API.